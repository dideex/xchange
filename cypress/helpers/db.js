const mongoose = require('mongoose')
const mongoConfig = require('./config.json').mongodb

module.exports = mongoose.connect(mongoConfig, {useNewUrlParser: true})
