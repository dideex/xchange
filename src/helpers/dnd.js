import React from 'react'
import {DragDropContext} from 'react-dnd'
import TestBackend from 'react-dnd-test-backend'

export function wrapInTestContext(DecoratedComponent) {
  const TestStub = props => <DecoratedComponent {...props} />
  return DragDropContext(TestBackend)(TestStub)
}
