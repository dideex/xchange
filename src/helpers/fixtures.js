export const fakeCurrnecy = [
  {
    order: 1,
    _id: 1,
    icon: 'eth',
    id: 'eth',
    name: 'tether',
    label: 'tether label',
    source: 'tether source wallet'
  },
  {
    order: 2,
    _id: 2,
    icon: 'bitcoin',
    id: 'bitcoin',
    name: 'lisk',
    label: 'lisk label',
    source: 'lisk source wallet'
  },
  {
    order: 3,
    _id: 3,
    icon: 'rur',
    id: 'rur',
    name: 'litecoin',
    label: 'litecoin label',
    source: 'litecoin source wallet'
  },
]


export const fakeData = {
  id: 'Test id',
  inputValue: '1000',
  toWallet: 'Test wallet',
  currencyInputLabel: 'Test input label',
  currencyOutputLabel: 'Test output label',
  fromWallet: 'Tests fromWallet',
  outputValue: '2000',
  email: 'Tests email',
  username: 'Tests username',
  paymentStatus: 1,
  loading: false,
  wallets: {
    eth: 'eth wallet',
    bitcoint: 'bitcoint wallet',
    rur: 'rur wallet',
  },
  login: 'Test login',
  updatePaymentStatus() {},
}

export const fakeUser = {
  username: 'Fake username',
  password: 'Fake password',
  email: 'fake@email.com',
  phone: '89145678901',
  fio: 'Fake fio',
  token: 'Fake token',
}

export const fakeBitcoinAddress = '3JQSigWTCHyBLRD979JWgEtWP5YiiFwcQB '
export const fakeCreditAddress = '1234 4321 5678 8765'

export const fakeWallets = {
  eth: 'Fake eth wallet',
  bitcoin: 'Fake bitcoin wallet',
  rur: 'Fake rur wallet'
}