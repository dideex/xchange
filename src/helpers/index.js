export {fakeCurrnecy} from './fixtures'

export const delay = (fn = () => {}, timeout = 0) =>
  new Promise(res =>
    setTimeout(() => {
      fn()
      res()
    }, timeout),
  )
