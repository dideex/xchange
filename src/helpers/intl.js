import React from 'react'
import {mount, shallow} from 'enzyme'
import {IntlProvider, intlShape} from 'react-intl'

import messages from '../locale/en.json'
const intlProvider = new IntlProvider({locale: 'en', messages}, {})
const {intl} = intlProvider.getChildContext()

function nodeWithIntlProp(node) {
  return React.cloneElement(node, {intl})
}

export function mountWithIntl(
  node,
  {context, childContextTypes, ...additionalOptions} = {},
) {
  return mount(nodeWithIntlProp(node), {
    context: Object.assign({}, context, {intl}),
    childContextTypes: Object.assign({}, {intl: intlShape}, childContextTypes),
    ...additionalOptions,
  })
}
