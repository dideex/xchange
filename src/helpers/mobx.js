import React from 'react'
import {Provider} from 'mobx-react'
import {lastOperations, cash, user} from '../store'

jest.mock('../components/Api', () => ({
  post: () => Promise.resolve({data: {}}),
  get: () => Promise.resolve({data: {}}),
  errorEmitter: jest.fn(data => fn => data(fn)),
}))

// jest.mock('../components/common', () => ({
//   noty: () => {},
//   Loading: () => <div>Loading</div>,
// }))

export const MobxProvider = ({children, ...stores}) => {
  return (
    <Provider
      lastOperationsStore={lastOperations}
      cashStore={cash}
      userStore={user}
      {...stores}
    >
      {children}
    </Provider>
  )
}
