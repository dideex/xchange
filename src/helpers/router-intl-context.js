import {mount} from 'enzyme'
import {BrowserRouter} from 'react-router-dom'
import {IntlProvider, intlShape} from 'react-intl'
import {shape} from 'prop-types'

import messages from '../locale/en.json'

export const testId = 'test id'

// Instantiate router context
const router = {
  history: {...new BrowserRouter().history},
  route: {
    location: {
      path: '/location',
      href: '/location',
    },
    match: {
      path: '/location',
      params: {
        id: testId,
      },
    },
  },
}

// const createContext = () => ({
//   context: {router},
//   childContextTypes: {router: shape({})},
// })

const intlProvider = new IntlProvider({locale: 'en', messages}, {})
const {intl} = intlProvider.getChildContext()

const createMixedContext = ({
  router,
  context,
  childContextTypes,
  ...additionalOptions
} = {}) => ({
  context: Object.assign({}, {router}, context, {intl}),
  childContextTypes: Object.assign(
    {},
    {router: shape({})},
    {intl: intlShape},
    childContextTypes,
  ),
  ...additionalOptions,
})

export function mountWrap(node) {
  return mount(node, createMixedContext({router}))
}

// export function shallowWrap(node) {
//   return shallow(node, createContext())
// }
