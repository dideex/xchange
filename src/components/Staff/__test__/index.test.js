import React from 'react'
import Component from '../index'
import {mountWithIntl} from '../../../helpers/intl'

describe('Staff index', () => {
  it('Basic markup', () => {
    const wrapper = mountWithIntl(<Component />)

    expect(wrapper.html()).toMatchSnapshot()
  })
})