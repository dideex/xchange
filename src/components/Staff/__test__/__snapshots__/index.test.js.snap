// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Staff index Basic markup 1`] = `"<section class=\\"css-uvb5el\\"><h2 class=\\"css-x184e\\"><span>Our team</span></h2><div class=\\"css-rdbcp7\\"><div class=\\"slick-slider slick-initialized\\" dir=\\"ltr\\"><svg style=\\"display: block; fill: #191716;\\" class=\\"chevron__left\\" x=\\"0px\\" y=\\"0px\\" viewBox=\\"0 0 400 400\\"><path style=\\"fill: #191716;\\" d=\\"M184.9,344.1L8.2,167.4c-8.5-8.5-8.5-22.3,0-30.9l20.6-20.6c8.5-8.5,22.3-8.5,30.8,0l140.7,140.1l140.7-140.1 c8.5-8.5,22.3-8.5,30.8,0l20.6,20.6c8.5,8.5,8.5,22.3,0,30.9L215.8,344.1C207.3,352.6,193.5,352.6,184.9,344.1L184.9,344.1z\\"></path></svg><div class=\\"slick-list\\"><div class=\\"slick-track\\" style=\\"opacity: 1;\\"><div data-index=\\"-2\\" tabindex=\\"-1\\" class=\\"slick-slide slick-cloned\\" aria-hidden=\\"true\\" style=\\"width: 0px;\\"><div><article class=\\"css-e6aqxp\\"><img src=\\"./img/staff3.png\\" alt=\\"Avatr\\" class=\\"css-1t6ic23\\"><svg viewBox=\\"0 0 291 167\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\"><path d=\\"M0.050293 166.311C13.0269 70.3545 0.050293 52.4758 84.9063 39.4629C125.755 48.968 164.572 49.3075 201.132 39.4629C308.556 55.8705 270.867 86.0832 290.05 166.311H0.050293Z\\" fill=\\"#211C1D\\"></path><path d=\\"M86.2604 39.4624L110.86 24.4126C133.766 53.3806 156.109 55.0779 177.887 24.4126L202.486 39.4624C205.871 50.8911 206.323 60.3962 202.486 66.8461L188.155 74.4276C193.91 80.4249 206.097 85.4037 202.486 89.4773C198.875 93.551 143.696 147.753 143.696 147.753C143.696 147.753 88.9686 94.5694 86.2604 90.8352C83.5522 87.1011 102.622 72.9566 102.622 72.9566C102.622 72.9566 90.9997 66.054 86.2604 61.9804C81.4082 58.02 86.2604 39.4624 86.2604 39.4624Z\\" fill=\\"#4D4B4C\\"></path><path d=\\"M115.26 33.5784V0.310547H174.502V34.71L145.727 60.7359L115.26 33.5784Z\\" fill=\\"#F1CBB9\\"></path><path d=\\"M110.86 24.4126L143.696 147.866L177.887 24.4126C177.887 24.4126 158.478 46.8175 145.727 46.8175C132.976 46.8175 110.86 24.4126 110.86 24.4126Z\\" fill=\\"white\\"></path><path d=\\"M145.727 46.7046L133.541 64.4701L138.618 77.8225L135.459 118.446L143.696 147.753L152.949 119.69L148.887 77.1436L155.996 65.4885L145.727 46.7046Z\\" fill=\\"#D86856\\"></path></svg><h3>Петухов В.М.</h3><p>Старший менеджер</p></article></div></div><div data-index=\\"-1\\" tabindex=\\"-1\\" class=\\"slick-slide slick-cloned\\" aria-hidden=\\"true\\" style=\\"width: 0px;\\"><div><article class=\\"css-e6aqxp\\"><img src=\\"./img/staff4.png\\" alt=\\"Avatr\\" class=\\"css-1t6ic23\\"><svg viewBox=\\"0 0 298 172\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\"><path d=\\"M115.105 21.0322V171.586H0.550049C0.550049 171.586 13.4119 69.9741 23.7495 53.139C34.2073 36.304 115.105 21.0322 115.105 21.0322Z\\" fill=\\"#4E3A2F\\"></path><path d=\\"M297.094 171.585H179.294V21.2725C187.708 22.956 259.35 37.6265 271.49 57.2273C284.232 78.0306 297.094 171.585 297.094 171.585Z\\" fill=\\"#4E3A2F\\"></path><path d=\\"M115.105 80.4354H179.294V0.228516H115.105V80.4354Z\\" fill=\\"#F1CBB9\\"></path><path d=\\"M115.105 44.3604C123.158 69.2522 171.961 69.2522 179.174 44.3604V171.585H115.105V44.3604Z\\" fill=\\"white\\"></path></svg><h3>Петухов В.М.</h3><p>Старший менеджер</p></article></div></div><div data-index=\\"0\\" class=\\"slick-slide slick-active slick-current\\" tabindex=\\"-1\\" aria-hidden=\\"false\\" style=\\"outline: none; width: 0px;\\"><div><article class=\\"css-e6aqxp\\"><img src=\\"./img/staff1.png\\" alt=\\"Avatr\\" class=\\"css-1t6ic23\\"><svg viewBox=\\"0 0 291 167\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\"><path d=\\"M0.050293 166.311C13.0269 70.3545 0.050293 52.4758 84.9063 39.4629C125.755 48.968 164.572 49.3075 201.132 39.4629C308.556 55.8705 270.867 86.0832 290.05 166.311H0.050293Z\\" fill=\\"#211C1D\\"></path><path d=\\"M86.2604 39.4624L110.86 24.4126C133.766 53.3806 156.109 55.0779 177.887 24.4126L202.486 39.4624C205.871 50.8911 206.323 60.3962 202.486 66.8461L188.155 74.4276C193.91 80.4249 206.097 85.4037 202.486 89.4773C198.875 93.551 143.696 147.753 143.696 147.753C143.696 147.753 88.9686 94.5694 86.2604 90.8352C83.5522 87.1011 102.622 72.9566 102.622 72.9566C102.622 72.9566 90.9997 66.054 86.2604 61.9804C81.4082 58.02 86.2604 39.4624 86.2604 39.4624Z\\" fill=\\"#4D4B4C\\"></path><path d=\\"M115.26 33.5784V0.310547H174.502V34.71L145.727 60.7359L115.26 33.5784Z\\" fill=\\"#F1CBB9\\"></path><path d=\\"M110.86 24.4126L143.696 147.866L177.887 24.4126C177.887 24.4126 158.478 46.8175 145.727 46.8175C132.976 46.8175 110.86 24.4126 110.86 24.4126Z\\" fill=\\"white\\"></path><path d=\\"M145.727 46.7046L133.541 64.4701L138.618 77.8225L135.459 118.446L143.696 147.753L152.949 119.69L148.887 77.1436L155.996 65.4885L145.727 46.7046Z\\" fill=\\"#D86856\\"></path></svg><h3>Абакумов М.А.</h3><p>Генеральный директор</p></article></div></div><div data-index=\\"1\\" class=\\"slick-slide slick-active\\" tabindex=\\"-1\\" aria-hidden=\\"false\\" style=\\"outline: none; width: 0px;\\"><div><article class=\\"css-e6aqxp\\"><img src=\\"./img/staff2.png\\" alt=\\"Avatr\\" class=\\"css-1t6ic23\\"><svg viewBox=\\"0 0 298 172\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\"><path d=\\"M115.105 21.0322V171.586H0.550049C0.550049 171.586 13.4119 69.9741 23.7495 53.139C34.2073 36.304 115.105 21.0322 115.105 21.0322Z\\" fill=\\"#4E3A2F\\"></path><path d=\\"M297.094 171.585H179.294V21.2725C187.708 22.956 259.35 37.6265 271.49 57.2273C284.232 78.0306 297.094 171.585 297.094 171.585Z\\" fill=\\"#4E3A2F\\"></path><path d=\\"M115.105 80.4354H179.294V0.228516H115.105V80.4354Z\\" fill=\\"#F1CBB9\\"></path><path d=\\"M115.105 44.3604C123.158 69.2522 171.961 69.2522 179.174 44.3604V171.585H115.105V44.3604Z\\" fill=\\"white\\"></path></svg><h3>Петухов В.М.</h3><p>Старший менеджер</p></article></div></div><div data-index=\\"2\\" class=\\"slick-slide\\" tabindex=\\"-1\\" aria-hidden=\\"true\\" style=\\"outline: none; width: 0px;\\"><div><article class=\\"css-e6aqxp\\"><img src=\\"./img/staff3.png\\" alt=\\"Avatr\\" class=\\"css-1t6ic23\\"><svg viewBox=\\"0 0 291 167\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\"><path d=\\"M0.050293 166.311C13.0269 70.3545 0.050293 52.4758 84.9063 39.4629C125.755 48.968 164.572 49.3075 201.132 39.4629C308.556 55.8705 270.867 86.0832 290.05 166.311H0.050293Z\\" fill=\\"#211C1D\\"></path><path d=\\"M86.2604 39.4624L110.86 24.4126C133.766 53.3806 156.109 55.0779 177.887 24.4126L202.486 39.4624C205.871 50.8911 206.323 60.3962 202.486 66.8461L188.155 74.4276C193.91 80.4249 206.097 85.4037 202.486 89.4773C198.875 93.551 143.696 147.753 143.696 147.753C143.696 147.753 88.9686 94.5694 86.2604 90.8352C83.5522 87.1011 102.622 72.9566 102.622 72.9566C102.622 72.9566 90.9997 66.054 86.2604 61.9804C81.4082 58.02 86.2604 39.4624 86.2604 39.4624Z\\" fill=\\"#4D4B4C\\"></path><path d=\\"M115.26 33.5784V0.310547H174.502V34.71L145.727 60.7359L115.26 33.5784Z\\" fill=\\"#F1CBB9\\"></path><path d=\\"M110.86 24.4126L143.696 147.866L177.887 24.4126C177.887 24.4126 158.478 46.8175 145.727 46.8175C132.976 46.8175 110.86 24.4126 110.86 24.4126Z\\" fill=\\"white\\"></path><path d=\\"M145.727 46.7046L133.541 64.4701L138.618 77.8225L135.459 118.446L143.696 147.753L152.949 119.69L148.887 77.1436L155.996 65.4885L145.727 46.7046Z\\" fill=\\"#D86856\\"></path></svg><h3>Петухов В.М.</h3><p>Старший менеджер</p></article></div></div><div data-index=\\"3\\" class=\\"slick-slide\\" tabindex=\\"-1\\" aria-hidden=\\"true\\" style=\\"outline: none; width: 0px;\\"><div><article class=\\"css-e6aqxp\\"><img src=\\"./img/staff4.png\\" alt=\\"Avatr\\" class=\\"css-1t6ic23\\"><svg viewBox=\\"0 0 298 172\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\"><path d=\\"M115.105 21.0322V171.586H0.550049C0.550049 171.586 13.4119 69.9741 23.7495 53.139C34.2073 36.304 115.105 21.0322 115.105 21.0322Z\\" fill=\\"#4E3A2F\\"></path><path d=\\"M297.094 171.585H179.294V21.2725C187.708 22.956 259.35 37.6265 271.49 57.2273C284.232 78.0306 297.094 171.585 297.094 171.585Z\\" fill=\\"#4E3A2F\\"></path><path d=\\"M115.105 80.4354H179.294V0.228516H115.105V80.4354Z\\" fill=\\"#F1CBB9\\"></path><path d=\\"M115.105 44.3604C123.158 69.2522 171.961 69.2522 179.174 44.3604V171.585H115.105V44.3604Z\\" fill=\\"white\\"></path></svg><h3>Петухов В.М.</h3><p>Старший менеджер</p></article></div></div><div data-index=\\"4\\" tabindex=\\"-1\\" class=\\"slick-slide slick-cloned\\" aria-hidden=\\"true\\" style=\\"width: 0px;\\"><div><article class=\\"css-e6aqxp\\"><img src=\\"./img/staff1.png\\" alt=\\"Avatr\\" class=\\"css-1t6ic23\\"><svg viewBox=\\"0 0 291 167\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\"><path d=\\"M0.050293 166.311C13.0269 70.3545 0.050293 52.4758 84.9063 39.4629C125.755 48.968 164.572 49.3075 201.132 39.4629C308.556 55.8705 270.867 86.0832 290.05 166.311H0.050293Z\\" fill=\\"#211C1D\\"></path><path d=\\"M86.2604 39.4624L110.86 24.4126C133.766 53.3806 156.109 55.0779 177.887 24.4126L202.486 39.4624C205.871 50.8911 206.323 60.3962 202.486 66.8461L188.155 74.4276C193.91 80.4249 206.097 85.4037 202.486 89.4773C198.875 93.551 143.696 147.753 143.696 147.753C143.696 147.753 88.9686 94.5694 86.2604 90.8352C83.5522 87.1011 102.622 72.9566 102.622 72.9566C102.622 72.9566 90.9997 66.054 86.2604 61.9804C81.4082 58.02 86.2604 39.4624 86.2604 39.4624Z\\" fill=\\"#4D4B4C\\"></path><path d=\\"M115.26 33.5784V0.310547H174.502V34.71L145.727 60.7359L115.26 33.5784Z\\" fill=\\"#F1CBB9\\"></path><path d=\\"M110.86 24.4126L143.696 147.866L177.887 24.4126C177.887 24.4126 158.478 46.8175 145.727 46.8175C132.976 46.8175 110.86 24.4126 110.86 24.4126Z\\" fill=\\"white\\"></path><path d=\\"M145.727 46.7046L133.541 64.4701L138.618 77.8225L135.459 118.446L143.696 147.753L152.949 119.69L148.887 77.1436L155.996 65.4885L145.727 46.7046Z\\" fill=\\"#D86856\\"></path></svg><h3>Абакумов М.А.</h3><p>Генеральный директор</p></article></div></div><div data-index=\\"5\\" tabindex=\\"-1\\" class=\\"slick-slide slick-cloned\\" aria-hidden=\\"true\\" style=\\"width: 0px;\\"><div><article class=\\"css-e6aqxp\\"><img src=\\"./img/staff2.png\\" alt=\\"Avatr\\" class=\\"css-1t6ic23\\"><svg viewBox=\\"0 0 298 172\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\"><path d=\\"M115.105 21.0322V171.586H0.550049C0.550049 171.586 13.4119 69.9741 23.7495 53.139C34.2073 36.304 115.105 21.0322 115.105 21.0322Z\\" fill=\\"#4E3A2F\\"></path><path d=\\"M297.094 171.585H179.294V21.2725C187.708 22.956 259.35 37.6265 271.49 57.2273C284.232 78.0306 297.094 171.585 297.094 171.585Z\\" fill=\\"#4E3A2F\\"></path><path d=\\"M115.105 80.4354H179.294V0.228516H115.105V80.4354Z\\" fill=\\"#F1CBB9\\"></path><path d=\\"M115.105 44.3604C123.158 69.2522 171.961 69.2522 179.174 44.3604V171.585H115.105V44.3604Z\\" fill=\\"white\\"></path></svg><h3>Петухов В.М.</h3><p>Старший менеджер</p></article></div></div><div data-index=\\"6\\" tabindex=\\"-1\\" class=\\"slick-slide slick-cloned\\" aria-hidden=\\"true\\" style=\\"width: 0px;\\"><div><article class=\\"css-e6aqxp\\"><img src=\\"./img/staff3.png\\" alt=\\"Avatr\\" class=\\"css-1t6ic23\\"><svg viewBox=\\"0 0 291 167\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\"><path d=\\"M0.050293 166.311C13.0269 70.3545 0.050293 52.4758 84.9063 39.4629C125.755 48.968 164.572 49.3075 201.132 39.4629C308.556 55.8705 270.867 86.0832 290.05 166.311H0.050293Z\\" fill=\\"#211C1D\\"></path><path d=\\"M86.2604 39.4624L110.86 24.4126C133.766 53.3806 156.109 55.0779 177.887 24.4126L202.486 39.4624C205.871 50.8911 206.323 60.3962 202.486 66.8461L188.155 74.4276C193.91 80.4249 206.097 85.4037 202.486 89.4773C198.875 93.551 143.696 147.753 143.696 147.753C143.696 147.753 88.9686 94.5694 86.2604 90.8352C83.5522 87.1011 102.622 72.9566 102.622 72.9566C102.622 72.9566 90.9997 66.054 86.2604 61.9804C81.4082 58.02 86.2604 39.4624 86.2604 39.4624Z\\" fill=\\"#4D4B4C\\"></path><path d=\\"M115.26 33.5784V0.310547H174.502V34.71L145.727 60.7359L115.26 33.5784Z\\" fill=\\"#F1CBB9\\"></path><path d=\\"M110.86 24.4126L143.696 147.866L177.887 24.4126C177.887 24.4126 158.478 46.8175 145.727 46.8175C132.976 46.8175 110.86 24.4126 110.86 24.4126Z\\" fill=\\"white\\"></path><path d=\\"M145.727 46.7046L133.541 64.4701L138.618 77.8225L135.459 118.446L143.696 147.753L152.949 119.69L148.887 77.1436L155.996 65.4885L145.727 46.7046Z\\" fill=\\"#D86856\\"></path></svg><h3>Петухов В.М.</h3><p>Старший менеджер</p></article></div></div><div data-index=\\"7\\" tabindex=\\"-1\\" class=\\"slick-slide slick-cloned\\" aria-hidden=\\"true\\" style=\\"width: 0px;\\"><div><article class=\\"css-e6aqxp\\"><img src=\\"./img/staff4.png\\" alt=\\"Avatr\\" class=\\"css-1t6ic23\\"><svg viewBox=\\"0 0 298 172\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\"><path d=\\"M115.105 21.0322V171.586H0.550049C0.550049 171.586 13.4119 69.9741 23.7495 53.139C34.2073 36.304 115.105 21.0322 115.105 21.0322Z\\" fill=\\"#4E3A2F\\"></path><path d=\\"M297.094 171.585H179.294V21.2725C187.708 22.956 259.35 37.6265 271.49 57.2273C284.232 78.0306 297.094 171.585 297.094 171.585Z\\" fill=\\"#4E3A2F\\"></path><path d=\\"M115.105 80.4354H179.294V0.228516H115.105V80.4354Z\\" fill=\\"#F1CBB9\\"></path><path d=\\"M115.105 44.3604C123.158 69.2522 171.961 69.2522 179.174 44.3604V171.585H115.105V44.3604Z\\" fill=\\"white\\"></path></svg><h3>Петухов В.М.</h3><p>Старший менеджер</p></article></div></div></div></div><svg style=\\"display: block; fill: #191716;\\" class=\\"chevron__right\\" x=\\"0px\\" y=\\"0px\\" viewBox=\\"0 0 400 400\\"><path style=\\"fill: #191716;\\" d=\\"M184.9,344.1L8.2,167.4c-8.5-8.5-8.5-22.3,0-30.9l20.6-20.6c8.5-8.5,22.3-8.5,30.8,0l140.7,140.1l140.7-140.1 c8.5-8.5,22.3-8.5,30.8,0l20.6,20.6c8.5,8.5,8.5,22.3,0,30.9L215.8,344.1C207.3,352.6,193.5,352.6,184.9,344.1L184.9,344.1z\\"></path></svg></div></div></section>"`;
